.github/workflows/main.yml

```diff
@@ -173,6 +178,43 @@ jobs:
     - name: run integration tests
       run: cd metagraph/build && make check
 
+  Build-and-Push-Docker:
+    # adapted from https://docs.github.com/en/actions/guides/publishing-docker-images#publishing-images-to-github-packages
+    if: github.ref == 'refs/heads/master'
+    needs: [Linux]
+    runs-on: ubuntu-latest
```

Comment:
Can we fix an LTS version, so this doesn't break if something changes in the update?